pkgs/fc/agent/fc/util/tests/test_enc.py

```diff
@@ -43,6 +43,7 @@ def test_initialize_enc_should_not_crash_when_initial_data_missing(
     assert log.has("initialize-enc-initial-data-not-found")
 
 
+@unittest.mock.patch("fc.util.enc.write_release_info")
```

Comment:
Is it possible to add some test cases with variations of valid but potentially plausible ENC json payloads?